projects/plugins/jetpack/extensions/blocks/map/map.php

```diff
@@ -101,7 +100,13 @@ function load_assets( $attr, $content ) {
 
 	Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
 
-	return preg_replace( '/<div /', '<div data-api-key="' . esc_attr( $access_token['key'] ) . '" ', $content, 1 );
+	$source = defined( 'IS_WPCOM' ) && IS_WPCOM ? 'mapkit' : 'mapbox';
+	// phpcs:ignore WordPress.Security.NonceVerification
```

Comment:
Why did we decide to ignore the the `WordPress.Security.NonceVerification` warning? I am sorry if I am being pesky. 🙈

Reply:
This is just a debug condition, the production code will lose this, and will read:

```php
if ( $source === 'mapkit' ) {
...
}
```
It seems a bit overkill to create a form with a nonce to enable this debug mode 😀